editor/editor_data.h

```diff
@@ -260,13 +270,19 @@ class EditorSelection : public Object {
 		return Object::cast_to<T>(selection[p_node]);
 	}
 
+	// Add an editor plugin which can provide metadata for selected nodes.
```

Comment:
Use present indicative tense for method descriptions:

```suggestion
	// Adds an editor plugin which can provide metadata for selected nodes.
```